source/agora/common/crypto/Key.d

```diff
@@ -196,6 +196,12 @@ public struct PublicKey
         dg(this.data[]);
     }
 
+    /// Support for deserialization
+    public void deserialize (scope DeserializeDg dg) @safe
+    {
+        this.data.opIndex[] = dg(this.data.opIndex.length)[];
```

Comment:
What does `opIndex[]` do? This is a new idiom to me..

Reply:
Oh. That's actually a bug in BitBlob's operator overload I need to fix.